Addon.Episerver.EnvironmentSynchronizer/Synchronizers/SiteDefinitions/SiteDefinitionSynchronizer.cs

```diff
@@ -110,5 +125,52 @@ private SiteDefinition GetExistingSiteDefinition(IEnumerable<SiteDefinition> exi
 
             return siteDefinition;
         }
+
+        private void SetForceLogin(SiteDefinition site, EnvironmentSynchronizerSiteDefinition siteDefinitionToUpdate)
```

Comment:
This feels like rather complex business logic that should be unit tested.

Reply:
Added unit tests for this logic